tests/run

```diff
@@ -0,0 +1,2 @@
+#! /usr/bin/env bash
+python -m unittest discover --start-directory tests
```

Comment:
I'd remove it, preferably we should be doing as little as possible to implement the tickets, and our tests should rigorously test that specification. That way the squashed commits form a succinct description of what was changed and why. This is also why it's nice to mention the ticket being closed in the merge commit.